### DISCUSSION TITLE
Cloudflared will not start

### DISCUSSION THREAD

**Discussion by Digita1Boy** [homeassistant-apps/app-cloudflared#592] [Q&A] [answered]
I've been using the addon in HA for a couple years without issue. It has suddenly stopped working and will not start with the following error

```
Error relocating /usr/lib/libonig.so.5: onig_check_callout_data_and_clear<: symbol not found
Error relocating /usr/lib/libonig.so.5: onig_get_callout_datt": symbol not found
```

I have not made any changes on the cloudflared side (addon or with the domain, cloudflare etc), or my with ha config. The only changes are typical ha version updates. Versions below.

Core 2024.1.6
Supervisor 2024.01.1
Operating System 11.5
Frontend 20240104.0 

I thought it may be a dependency issue, but then would have expected a lot of other people with issues. I've searched, and it's just me... Any ideas? thx

**Comment by elcajon** [comment]
What system architecture is your Home Assistant OS running on? The error message looks like the add-on image is not compatible with your system.

**Comment by Digita1Boy** [comment]
Running in a VMBox. But nothing has changed there - been stable for 2 years. The only change is a recent Cloudfared add-on update, and it just stopped working. It's been rock solid for 2 years, so don't see how it can be system incompatibility. I've just updated to 2024.2.1 to check if something was updated, but same issue. thx

**Comment by Digita1Boy** [comment, marked as answer]
Just reporting back that it is working again. I have updated to HA 2024.2.2 and CF 5.1.4. One of those updates fixed the issue in case others come searching here for solutions. thx
